test/textinput.test.tsx

```diff
@@ -13,15 +13,16 @@ describe('TextInput', () => {
 
   it('renders a TextInput with attributes', () => {
     const name = 'test_input'
-    const value = 'Testing the text input component'
-    const wrapper = mount(<TextInput name={name} value={value} />)
+    const defaultValue = 'Testing the text input component defaultValue'
```

Comment:
There seems to be a conflict over this const

Reply:
Sorry for the delays, I will only be able to volunteer on weekends and not much considering I am having a lot of responsablity at work and it's my first couple of months. Sorry for the inconvenience and when I get some free time I will return to contributing but if you guys want this functionality asap feel free to take it over and do as you wish. 
I'll take a look at this branch and the other 18n translations tommorow and if I feel I won\t be able to get the work done I'll leave notice. Pardon the incovenience and late reply